AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -321,6 +340,7 @@ - (void)acquireTokenSilentlyWithMSALAccount:(MSALAccount *)account {
                       [MSAuthTokenContext sharedInstance].authToken = result.idToken;
                       [strongSelf saveAuthToken:result.idToken];
                       [strongSelf saveAccountId:(NSString * _Nonnull) result.account.homeAccountId.identifier];
+                      MSLogInfo([MSIdentity logTag], @"User sign-in succeeded.");
```

Comment:
It is more like to refresh token so you need to differentiate this log message from interactive sign-in below.